importer/SeBbrSv.py

```diff
@@ -356,6 +379,9 @@ def __init__(self, db_row_dict, mapping, data_files, existing, repository):
         self.set_monuments_all_id()
         self.set_changed()
         self.wlm_source = self.create_wlm_source(self.monuments_all_id)
+        self.create_bbr_link()
+        self.set_bbr()
+        self.get_online_data()
```

Comment:
doesn't this need to come before `create_bbr_link()` to ensure the data has been downloaded.